docker-compose.yml

```diff
@@ -53,6 +54,49 @@ services:
       - prover
       - proposer
 
+  l2_execution_engine:
+    image: nethswitchboard/taiko-geth:surge
```

Comment:
If we keep it in single file than the instruction will be simpler, because we just run `docker compose --profile proposer --profile prover --profile blockscout up -d` and all is up. If we don't want a prover then `docker compose --profile proposer --profile blockscout up -d`
WDYT?